agent/osv_file_handler.py

```diff
@@ -103,6 +82,10 @@ def parse_results(output: str) -> Iterator[Vulnerability]:
     """
 
     data = json.loads(output, strict=False)
+    if data.get("results") is None:
+        logger.info("Osv returns null result.")
+        return
```

Comment:
return is `return None`, ur method expecting to return iterator, & when u use it in emit results with:  `for vuln in osv_file_handler.parse_results(output):` that will crash!